[2025-08-23T21:00:03.275-0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-23T21:00:03.296-0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_processamento_viagens.executar_etl_de_viagens scheduled__2025-08-23T23:50:00+00:00 [queued]>
[2025-08-23T21:00:03.299-0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_processamento_viagens.executar_etl_de_viagens scheduled__2025-08-23T23:50:00+00:00 [queued]>
[2025-08-23T21:00:03.299-0300] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-23T21:00:03.307-0300] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): executar_etl_de_viagens> on 2025-08-23 23:50:00+00:00
[2025-08-23T21:00:03.314-0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline_processamento_viagens', 'executar_etl_de_viagens', 'scheduled__2025-08-23T23:50:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/dag_processamento_viagens.py', '--cfg-path', '/tmp/tmpng898v_w']
[2025-08-23T21:00:03.316-0300] {standard_task_runner.py:91} INFO - Job 15: Subtask executar_etl_de_viagens
[2025-08-23T21:00:03.317-0300] {logging_mixin.py:188} WARNING - /home/caioa/Airflow/venv/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=5484) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-23T21:00:03.317-0300] {standard_task_runner.py:63} INFO - Started process 5498 to run task
[2025-08-23T21:00:03.344-0300] {task_command.py:426} INFO - Running <TaskInstance: pipeline_processamento_viagens.executar_etl_de_viagens scheduled__2025-08-23T23:50:00+00:00 [running]> on host PCzo.localdomain
[2025-08-23T21:00:03.380-0300] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='pipeline_processamento_viagens' AIRFLOW_CTX_TASK_ID='executar_etl_de_viagens' AIRFLOW_CTX_EXECUTION_DATE='2025-08-23T23:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-23T23:50:00+00:00'
[2025-08-23T21:00:03.381-0300] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-23T21:00:03.393-0300] {logging_mixin.py:188} INFO - --- Iniciando Pipeline de ETL de Viagens ---
[2025-08-23T21:00:03.394-0300] {logging_mixin.py:188} INFO - Arquivo de origem não encontrado ou vazio: /home/caioa/Airflow/data/novas_avaliacoes.csv
[2025-08-23T21:00:03.394-0300] {logging_mixin.py:188} INFO - Nenhum dado novo para processar.
[2025-08-23T21:00:03.394-0300] {logging_mixin.py:188} INFO - --- Pipeline de ETL de Viagens Concluído ---
[2025-08-23T21:00:03.394-0300] {python.py:237} INFO - Done. Returned value was: None
[2025-08-23T21:00:03.394-0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-23T21:00:03.396-0300] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline_processamento_viagens, task_id=executar_etl_de_viagens, run_id=scheduled__2025-08-23T23:50:00+00:00, execution_date=20250823T235000, start_date=20250824T000003, end_date=20250824T000003
[2025-08-23T21:00:03.410-0300] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-23T21:00:03.416-0300] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-23T21:00:03.417-0300] {local_task_job_runner.py:222} INFO - ::endgroup::
